qemu/tests/cfg/virtio_port_hotplug.cfg

```diff
@@ -33,3 +33,17 @@
                 clean_cmd = del /f /q
                 tmp_dir = %TEMP%
                 python_bin = python2.7
+        - hotplug_various_chardev:
+            type = virtio_serial_various_chardev_hotplug
+            serials = ""
+            start_vm = yes
+            extra_chardevs = "channel1 channel2 channel3 channel4 channel5 channel6"
+            chardev_backend_channel2 = tcp_socket
+            chardev_backend_channel3 = pty
+            chardev_backend_channel4 = null
+            chardev_backend_channel5 = file
+            chardev_backend_channel6 = ringbuf
+            ringbuf_write_size = 2048
+            ringbuf_read_size_channel6 = 1000
+            ringbuf_data_channel6 = 'adcdefgh'
```

Comment:
Not necessary to have suffix `_channel6` for `ringbuf_read_size` and `ringbuf_data`, that is really confusing, cause we use the suffix to distinguish the options when create different devices, here this two is only for specific test steps, have nothing to do with device name.

Reply:
> Not necessary to have suffix `_channel6` for `ringbuf_read_size` and `ringbuf_data`, that is really confusing, cause we use the suffix to distinguish the options when create different devices, here this two is only for specific test steps, have nothing to do with device name.

They are the options of chardev ringbuf, but there is only one chardev ringbuf, so it is not necessary, If you need to test different data with different ringbuf devices, seems we need the suffix `_channel6`

Reply:
No, they are the options of the qmp command, not chardev device, if so, you should add those options here: https://github.com/avocado-framework/avocado-vt/pull/2239/files#diff-4045499c9c61cb3d07734c53a7dcbb26R1970

Reply:
Even you have only one device, you can send different data with read with different sizes several times by the qmp command, that's the difference. You don't even need define them in cfg files, they can be variants only defined in your test scripts.

Reply:
> Even you have only one device, you can send different data and read with different sizes several times by the qmp command, that's the difference. You don't even need define them in cfg files, they can be variants only defined in your test scripts.

Got it and updated, please help to review this, thanks.